models/stg_zendesk.yml

```diff
@@ -281,33 +363,53 @@ models:
         description: Year in which daylight savings occurred. 
       - name: daylight_offset_minutes
         description: Number of **minutes** added during Daylight Savings Time. 
+      - name: source_relation
+        description: >
+          The schema or database this record came from if you are unioning multiple Zendesk connectors together in this package.
+          Empty string if you are not.
 
   - name: stg_zendesk__time_zone
     description: Offsets (from UTC) for each timezone. 
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns: 
+            - time_zone
+            - source_relation 
```

Comment:
This test is resulting in a failure when running on the `zendesk_yashwanth` schema. Do you have any understanding why this is failing?

![image](https://github.com/fivetran/dbt_zendesk_source/assets/74217849/6ebd4065-5833-48e1-b28f-6d3c716f60d9)

Reply:
It looks like this is the result of the `time_zone` name creating a struct when we don't want it to. I recall you had a fix for this. Do you know if that may have been missed for some reason?

![image](https://github.com/fivetran/dbt_zendesk_source/assets/74217849/ab77ef02-c5ac-4b4e-b25e-3c95c956cd54)

Reply:
Could it be that we need to add the alias to the union_data macro [here](https://github.com/fivetran/dbt_fivetran_utils/blob/c8ace80ff5d59a458821a5b078dc745093eba175/macros/union_data.sql#L129).

So it would be:
```sql
{%- if table_exists -%}
    select
        {{ dbt_utils.star(from=relation.value) }}
    from {{ relation.value }} as {{ table_identifier }}_table
```

Reply:
ah yes i missed adding this to the single-connector use-case inside `union_data`